src/lib/Containers/Conversation.tsx

```diff
@@ -64,14 +81,20 @@ interface Props extends RelayProps {
 
 interface State {
   sendingMessage: boolean
+  isConnected: boolean
 }
 
 export class Conversation extends React.Component<Props, State> {
   constructor(props) {
     super(props)
+
+    // Assume if the component loads, connection exists (this way the banner won't flash unnecessarily)
     this.state = {
       sendingMessage: false,
+      isConnected: true,
     }
+
+    this.handleConnectivityChange = this.handleConnectivityChange.bind(this)
```

Comment:
It may well be fixed by now, indeed, but let’s verify that before using it again.